doc/releases/changelog-dev.md

```diff
@@ -4,6 +4,29 @@
 
 <h3>New features since last release</h3>
 
+* Many parametrized operations now have the attribute `ndim_params` and
+  allow arguments with a broadcasting dimension. Also see entries for #2590
+  and #2575 below for details.
+  [(#2609)](https://github.com/PennyLaneAI/pennylane/pull/2609)
```

Comment:
Based on this sentence, I would combine all three entries into a singular entry?